src/cmd/sanity_test/export/sharepoint.go

```diff
@@ -14,15 +20,65 @@ func CheckSharePointExport(
 	ac api.Client,
 	envs common.Envs,
 ) {
-	drive, err := ac.Sites().GetDefaultDrive(ctx, envs.SiteID)
-	if err != nil {
-		common.Fatal(ctx, "getting the drive:", err)
+	if envs.Category == "lists" {
+		CheckSharepointListsExport(ctx, ac, envs)
+	}
+
+	if envs.Category == "libraries" {
+		drive, err := ac.Sites().GetDefaultDrive(ctx, envs.SiteID)
+		if err != nil {
+			common.Fatal(ctx, "getting the drive:", err)
+		}
+
+		envs.RestoreContainer = filepath.Join(envs.RestoreContainer, "Libraries/Documents") // check in default loc
+		driveish.CheckExport(
+			ctx,
+			ac,
+			drive,
+			envs)
+	}
+}
+
+func CheckSharepointListsExport(
+	ctx context.Context,
+	ac api.Client,
+	envs common.Envs,
+) {
+	exportFolderName := "Lists"
```

Comment:
The `.Human()` value of the lists category would work better here than a hardcoded string.